randovania/games/prime3/logic_database/Bryyo - Reptilicus.txt

```diff
@@ -634,7 +635,10 @@ Extra - asset_id: 6613311822671481119
   > Pickup (Missile Expansion)
       Trivial
   > Event - Leviathan Shield
-      After Jungle Generator Ship Missile Attempt and After North Jungle Court Turret and Use Ship Missile
+      All of the following:
+          After Jungle Generator Ship Missile Attempt and After North Jungle Court Turret and Use Ship Missile
+          # Get to the ledge at Field Access, then return https://youtu.be/I5LtCOvlR80
+          After South Jungle Court Turret or Knowledge (Beginner)
```

Comment:
Not sure I like this, due to 2 reasons:
1. needing to transition through a room
2. the comment of 
> if you go too far and the room unloads, this won't work.

which i'm interpreting as "one chance only, if you fuck it up,good luck"

Reply:
Not sure why transitionning to another room is a problem here? It's not OoB, and I'm fairly certain there isn't much else you can do...
Would something like this https://www.youtube.com/watch?v=03I2HiDZoCg&t=9865s be better as a video? It doesn't technically transition rooms more than once, bit it won't show that just walking into the room won't work and that it can be fixed if you do ; imo it's even worse than my previous wording when it comes to implying this only works once

Reply:
Well, after looking at the room in PWE, I don't think it's about an invisible trigger at all. In fact, the ship command icon to effectively bomb the generator is activated by a counter, which itself is linked to a special function for each turret. If the corresponding turret is destroyed, then it will increment the counter by 1. The counter needs to reach 2 for the bomb run icon to appear.
My theory is that each time a destroyed turret is rendered, the special function returns true, and increments the counter. As such, if we leave without unloading the room, the counter will remain at 1, but the destroyed turret will be culled, which does not decrement the counter because of an oversight. Then, re-entering the room will render the destroyed turret again, which will increment the counter to 2 and activate the bomb run icon.
If my theory is correct, then we should also be able to perform the same manip when destroying the South Jungle Court turret. I'll test it out a later today and modify the video title and description if my theory turns out to be relevant.

Also, the only misc resource available for corruption is "Skipping Energy Cells", so I don't think it's the relevant kind of resource to put, so how should I go about adding a new kind of misc resource, and what name should I give it? Something like "Can open door" ?

Reply:
> how should I go about adding a new kind of misc resource

You click on "new..."

> what name should I give it

Entrance Rando, then have a required disabled entrance rando resource.


I would also still bump the knowledge requirement because "you can permanently lock yourself out by accidentally unloading the room and then saving" does not sound like beginner.

Reply:
I may not have insisted enough, but this isn't a one shot trick at all. I've modified my description to this, if it's clearer that you can try again if you mess it up : 
> Please note that this isn't a room reload, if you go too far and the room unloads, you will still not be able to see the icon, and will have to start the manip again

Reply:
ahh ok